public/.well-known/security.txt

```diff
@@ -0,0 +1,2 @@
+Contact: https://crates.io/policies/security
+Expires: 2025-06-25T00:00:00.000Z
```

Comment:
We're 100% going to forget this, aren't we?

I wonder if we should have a scheduled GitHub action that monitors this and tells us if it's close. (I can write that after this gets merged.)

Reply:
hmm, that's a good point. I guess alternatively the file is small enough that we could also generate it on startup and serve it from the axum server 🤔

Reply:
well the point is that a person should look at it every once in a while.........